app/views/layouts/application.html.erb

```diff
@@ -19,10 +19,18 @@
         )
     %>
     <% if flash[:notice] %>
-      <div class="alert alert-warning">
+      <div class="alert alert-success" id="alert-message">
         <%= flash[:notice] %>
       </div>
     <% end %>
+
+    <% if flash[:alert] %>
+      <div class='alert alert-danger' id="alert-message">
+          <%= flash[:alert] %>
+      </div>
+    <% end %>
+
+
```

Comment:
These styles will not exist when my PR goes through. Wanna create your own alerts or want me to?